chart/env/prod.yaml

```
@@ -537,3 +541,31 @@ workers:
       limits:
         cpu: 2
         memory: "1Gi"
+
+webhook:
+  # Number of uvicorn workers for running the application
+  # (2 x $num_cores) + 1
+  # https://docs.gunicorn.org/en/stable/design.html#how-many-workers
+  uvicornNumWorkers: "9"
+  nodeSelector:
+    role-datasets-server-webhook: "true"
+  tolerations:
+    - key: "huggingface.co/datasets-server-webhook"
+      operator: "Exists"
+      effect: "NoSchedule"
+  replicas: 20
+  service:
+    type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/target-node-labels: role-datasets-server-webhook=true
```

missing group.order: 

```suggestion
      alb.ingress.kubernetes.io/group.order: "5"
      alb.ingress.kubernetes.io/target-node-labels: role-datasets-server-webhook=true
```